src/control/system/raft/database_test.go

```diff
@@ -144,9 +144,9 @@ func TestSystem_Database_Cancel(t *testing.T) {
 		return nil
 	})
 
-	waitForLeadership(ctx, t, db, true, 15*time.Second)
+	waitForLeadership(ctx, t, db, true, 30*time.Second)
```

Comment:
Just wondering if this could be redesigned to eliminate timeouts if they're not really a part of what's being tested.

Reply:
Yeah that's probably best. I was hedging a bit, thinking that maybe we'd still want the ability to check that things happen in a reasonable amount of time, but it's certainly clearer to just remove them completely.